packages/amplify-nodejs-function-template-provider/amplify-plugin.json

```diff
@@ -29,6 +29,10 @@
       {
         "name": "GraphQL Lambda Authorizer",
         "value": "lambda-auth"
+      },
+      {
+        "name": "AppSync Todo",
```

Comment:
Since this changes the customer facing prompts, it might need an API review. Can you quickly check with Rene if the naming here is okay?